### PR TITLE
Update geometry_helper.gd and hide warnings (unused signals etc.)

### PR DESCRIPTION
**Pull request by kopeek**
added a function to check if a polygon is valid

## Description

Add your description here, especially why you made these changes and what problem you tried to solve.

## Addressed issues

- List issues here, reference them via #

## Screenshots (Optional)

Show your changes with screenshots

### CODE REVIEW

File: autoload/general/game_globals.gd
Comment (sempitern0) [inline autoload/general/game_globals.gd:1]: I think we can avoid conversion to float using `clampi()` instead and remove the explicit warning as this function should only work with integers.

Comment (kopeek) [reply autoload/general/game_globals.gd:1]: Just remove warnings and update Geometry_helper.gd

**Comment by sempitern0** [comment]
Hi @kopeek I don't see anything wrong with this new functions but it would be nice to know if you can provide examples or use cases for this methods.

And I don't understand very well the addition inside the `chunk `method in the `ArrayHelper`, what does this conditional do exactly?

Cheers

**Comment by kopeek** [comment]
Hello @ninetailsrabbit
1. No need for this really, I can but it just does that it says.
2. I saw this somewhere and I Implemented this, it says what it does (in simple terms)
When you will get result like this [[1,2,3], [4,5,6], [7,8]] you see that the last array isn't full (the function was called with variable count = 3), it will just remove that last, if you only want to have only full arrays

**Pull request merged**